packages/config/config/devices/0x0312/z97.json

```diff
@@ -0,0 +1,95 @@
+// NIE Technology Co., Ltd. ZW97 / Eva Logik
+// Outdoor Z-Wave Plug-In Module (2-Channel)
+// TODO: ZW97 support Over temperature protection, When the temperature is greater than 105 degrees Celsius, the Heat Alarm signal is sent out Notification Type : Heat Alarm 0x04 . Event :Overheat detected 0x02
```

Comment:
It's inappropriate to leave a TODO at the top of the file like this. Please remove it. If an alarm goes off it will be passed through automatically anyway.

Reply:
OK